src/Map/index.tsx

```diff
@@ -156,6 +159,19 @@ const Map: React.FC<Props> = (props) => {
     geojsonSource?.setData(sourceJson as FeatureCollection)
   }, [sourceJson])
 
+  // Use to travel to location on card click
+  useEffect(() => {
+    const map = mapRef.current
+    if (firstRender) {
+      setFirstRender(false)
+    } else if (map != null && props.pointOptions?.zoomLocation != null) {
+      map.easeTo({
+        center: props.pointOptions?.zoomLocation as LngLatLike,
+        zoom: 9,
```

Comment:
could be `zoom: onClickZoomIn` so the zoom level of clicking a card matches the zoom level of clicking a point